docs/source/guide/clients.rst

```diff
@@ -132,6 +132,12 @@ which may interact with boto3’s client. The majority of users will not
 need to use these interfaces, but those that do should no longer
 consider their clients thread-safe without careful review.
 
+.. note::
+    `boto3.client('<client_name>')` is an alias for creating a client with a
```

Comment:
Using a single backtick will result in italicized text instead of inline-code. Also I think this should be `service_name` and not `client_name`.

```suggestion
    ``boto3.client('<service_name>')`` is an alias for creating a client with a
```